src/redux/books/booksSlice.js

```diff
@@ -1,28 +1,55 @@
+/* eslint-disable no-param-reassign */
```

Comment:
The linters are there to help you provide the best and up to standard code. So kindly do not disable them. Please consider removing this line and work within the constraints of the linters :+1: